vcd/resource_vcd_catalog_media.go

```diff
@@ -132,39 +171,74 @@ func resourceVcdMediaCreate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error waiting for task to complete: %+v", err)
 	}
 
-	d.SetId(catalogName + ":" + mediaName)
-
 	log.Printf("[TRACE] Catalog media created: %#v", mediaName)
 
 	err = createOrUpdateMediaItemMetadata(d, meta)
 	if err != nil {
 		return fmt.Errorf("error adding media item metadata: %s", err)
 	}
 
+	//sets Id in findCatalogItem func - there isn't ID with media type
```

Comment:
Is this comment obsolete? There is an ID in media type

Reply:
yes, removed